saleor/graphql/order/types.py

```diff
@@ -657,6 +659,13 @@ class Order(CountableDjangoObjectType):
     is_shipping_required = graphene.Boolean(
         description="Returns True, if order requires shipping.", required=True
     )
+    is_click_and_collect = graphene.Boolean(
+        description="Returns True, if order is set as click and collect"
```

Comment:
"order is set as click and collect" - this could be more accurate, for example "Returns true if Click & Collect is selected as a delivery method for this order." Also, we should be consistent when referring to this feature in descriptions, docs, etc. Here we have "click and collect", below we have "C&C".

Reply:
Done.